lock_unlock_feature.ino

```diff
@@ -0,0 +1,70 @@
+#include <Keyboard.h>
+#include <SPI.h>
+#include <MFRC522.h>
+
+#define SS_PIN 10
+#define RST_PIN 5
+#define KEY_RETURN 0xB0  // The hex value for the return key is 0xB0.
+
+MFRC522 mfrc522(SS_PIN, RST_PIN);
+char Enter = KEY_RETURN;  // Return key is declared as Enter.
+String readid;
+String card1 = "3A3D9D9E";  // Change this value to the UID of your card.
+bool screenLocked = false;
+
+void setup() {
+  Serial.begin(9600);
+  Keyboard.begin();
+  SPI.begin();
+  mfrc522.PCD_Init();
+  Serial.println("Setup complete. Scan your RFID tag.");
+}
+
+void temp(byte *buffer, byte bufferSize) {
+  // Function to store card UID as a string datatype.
+  readid = "";
+  for (byte i = 0; i < bufferSize; i++) {
+    readid = readid + String(buffer[i], HEX);
+  }
+  readid.toUpperCase(); // Ensure the UID is in uppercase
+  Serial.print("Card UID: ");
+  Serial.println(readid);
+}
+
+void lockScreen() {
+  Serial.println("Card recognized. Locking screen.");
+  Keyboard.press(KEY_LEFT_GUI);  // Press the left Windows key.
+  Keyboard.press('l');           // Press the "l" key.
+  Keyboard.releaseAll();
+  delay(200);         // Release all keys.
+  screenLocked = true;
+}
+
+void unlockScreen() {
+  Serial.println("Card recognized. Unlocking screen.");
+  Keyboard.print("23990");       // Change this value to your Windows PIN/Password.
```

Comment:
The password should not be in clear text. This is a big security issue.  Also, it should be able to store multiple passwords for multiple cards and users.